where-to-travel/src/main/webapp/script.js

```diff
@@ -411,9 +411,9 @@ function getLocationCardHtml(title, address, directionsLink, timeStr) {
  */
 function getLoginHtml() {
   return `<img onclick="showInfoModal()" class="btn btn-icon" src="icons/help.svg">
-          <a class="btn btn-outline-primary" style="text-align: center" href="login.html">Login</a>
+          <a class="btn btn-outline-primary btn-small-padding" style="text-align: center" href="login.html">Login</a>
```

Comment:
When we merge with master, we need to be careful about replacing hrefs with the onclicks and the corresponding functions. I believe a future pull request will move these html strings into separate files to mitigate the problem.

Reply:
Good point